conf/messages

```diff
@@ -2241,7 +2241,8 @@ responsiblepeople.fit_and_proper.notice.text2 = The fit and proper test is for m
 responsiblepeople.fit_and_proper.notice.heading2 = HMRC’s approval check
 responsiblepeople.fit_and_proper.notice.text3 = The approval check is for accountancy service providers, art market participants, estate agent businesses, and high value dealers.
 
-responsiblepeople.fit_and_proper.heading = Has {0} passed HMRC’s fit and proper test?
+responsiblepeople.fit_and_proper.heading = HMRC fit and proper test
```

Comment:
I think because the h1 has changed, the page title should also change to match. https://design.tax.service.gov.uk/hmrc-design-patterns/page-title/